src/components/Admin/Admin.test.jsx

```diff
@@ -294,4 +295,222 @@ describe('<Admin />', () => {
+          }}
+          match={{
+            url: '/',
+            params: {
+              actionSlug: 'enrolled-learners',
+            },
+          }}
+        />
+      ));
+      wrapper.find('.download-btn').simulate('click');
+      expect(spy).toHaveBeenCalled();
+    });
+
+    it('enrolled learners inactive courses', () => {
+      spy = jest.spyOn(EnterpriseDataApiService, 'fetchEnrolledLearnersForInactiveCourses');
+      const wrapper = mount((
+        <AdminWrapper
+          enterpriseId="test-enterprise-id"
+          table={{
+            'enrolled-learners-inactive-courses': {
+              data: {
+                results: [{ id: 1 }],
+              },
+            },
+          }}
+          match={{
+            url: '/',
+            params: {
+              actionSlug: 'enrolled-learners-inactive-courses',
+            },
+          }}
+        />
+      ));
+      wrapper.find('.download-btn').simulate('click');
+      expect(spy).toHaveBeenCalled();
+    });
+
+    it('learners active week', () => {
+      spy = jest.spyOn(EnterpriseDataApiService, 'fetchCourseEnrollments');
+      const wrapper = mount((
+        <AdminWrapper
+          enterpriseId="test-enterprise-id"
+          table={{
+            'learners-active-week': {
+              data: {
+                results: [{ id: 1 }],
+              },
+            },
+          }}
+          match={{
+            url: '/',
+            params: {
+              actionSlug: 'learners-active-week',
+            },
+          }}
+        />
+      ));
+      wrapper.find('.download-btn').simulate('click');
+      expect(spy).toHaveBeenCalled();
+    });
+
+    it('learners inactive week', () => {
+      spy = jest.spyOn(EnterpriseDataApiService, 'fetchCourseEnrollments');
+      const wrapper = mount((
+        <AdminWrapper
+          enterpriseId="test-enterprise-id"
+          table={{
+            'learners-inactive-week': {
+              data: {
+                results: [{ id: 1 }],
+              },
+            },
+          }}
+          match={{
+            url: '/',
+            params: {
+              actionSlug: 'learners-inactive-week',
+            },
+          }}
+        />
+      ));
+      wrapper.find('.download-btn').simulate('click');
+      expect(spy).toHaveBeenCalled();
+    });
+
+    it('learners inactive month', () => {
+      spy = jest.spyOn(EnterpriseDataApiService, 'fetchCourseEnrollments');
+      const wrapper = mount((
+        <AdminWrapper
+          enterpriseId="test-enterprise-id"
+          table={{
+            'learners-inactive-month': {
+              data: {
+                results: [{ id: 1 }],
+              },
+            },
+          }}
+          match={{
+            url: '/',
+            params: {
+              actionSlug: 'learners-inactive-month',
+            },
+          }}
+        />
+      ));
+      wrapper.find('.download-btn').simulate('click');
+      expect(spy).toHaveBeenCalled();
```

Comment:
May want to make sure these are calledWith the proper parameters as fetchCourseEnrollments gets called multiple times... Could also make sure we're passing `{ csv: true }`

Reply:
That's probably a good idea to check that the proper parameters are being passed. Though, I think because we have `spy.mockRestore()` in the `afterEach()`, checking `toHaveBeenCalled()` for `fetchCourseEnrollments` each time it's called should check that it's being called each time.

I will change this to `toHaveBeenCalledWith` and check the params.

Reply:
Sorry, I meant its called from different tests/scenarios with different parameters. You got it :)